vtex/address-form
# CHK-788: Accept geolocation rules in validate function

#### What is the purpose of this pull request?

This PR updates the `isValidAddress` function to accept the rules definition inside `rules.geolocation`, so we can properly validate the address in geolocation scenario inside Checkout.

#### What problem is this solving?

Fixes an issue where the number field wasn't being validated for Argentinian stores.

#### How should this be manually tested?

[Workspace](https://lucas2--vtexgame1geo.myvtex.com/checkout/cart/add/?sku=312&qty=1&seller=1&sc=2)

1. Access the above workspace
2. Go to shipping
3. Use the address `guatemala` and `Argentina` country, select the first 
   autocompleted result.
4. You should not be able to proceed to payment without typing in the number 
   field. Moreover, you should see a message indicating that the number is 
   required to proceed.

#### Screenshots or example usage

#### Types of changes
